support/ebpf/off_cpu.ebpf.c

```diff
@@ -0,0 +1,86 @@
+#include "bpfdefs.h"
+#include "types.h"
+#include "tracemgmt.h"
+
+// kprobe_progs maps from a program ID to a kprobe eBPF program
+bpf_map_def SEC("maps") kprobe_progs = {
+  .type = BPF_MAP_TYPE_PROG_ARRAY,
+  .key_size = sizeof(u32),
+  .value_size = sizeof(u32),
+  .max_entries = NUM_TRACER_PROGS,
+};
+
+// profile_off_cpu communicates scheduler tasks.
+bpf_map_def SEC("maps") profile_off_cpu = {
```

Comment:
`profile_off_cpu` is too generic / doesn't convey enough information but also confusing me:

```suggestion
// sched_times keeps track of sched_switch call times.
bpf_map_def SEC("maps") sched_times = {
```